include/interaction/ArcBallCamera.h

```diff
@@ -1,25 +1,25 @@
 //  ----------------------------------
-//  Copyright � 2017, Brown University, Providence, RI.
-//  
+//  Copyright � 2017, Brown University, Providence, RI.
```

Comment:
Looks like something bugged out and the copyright symbol stopped being recognized
```suggestion
//  Copyright © 2017, Brown University, Providence, RI.
```

Reply:
done